prettier.config.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * See https://prettier.io/docs/en/options.html.
+ */
+module.exports = {
+  printWidth: 80,
+  semi: true,
+  singleQuote: true,
+  jsxSingleQuote: false,
+  quoteProps: 'consistent',
+  trailingComma: 'all',
+  bracketSpacing: true,
+  bracketSameLine: false,
+  arrowParens: 'always',
+  proseWrap: 'preserve',
+  htmlWhitespaceSensitivity: 'css',
+  endOfLine: 'lf',
+  embeddedLanguageFormatting: 'auto',
```

Comment:
minor nitpick, can this be alpha sorted? or maybe sorted by the options in the docs (if alpha sorting does not make sense)?